src/ctia/task/migration/store.clj

```diff
@@ -705,11 +706,8 @@ when confirm? is true, it stores this state and creates the target indices."
 (s/defn setup!
   "setup store service"
   [{{:keys [get-config]} :ConfigService :as services} :- MigrationStoreServices]
-  ;; TODO will be rolled into trapperkeeper bootstrap
```

Comment:
Starting an app here turned out not to be necessary, so this impl doesn't need to change.